datatune/core/map.py

```diff
@@ -72,16 +73,19 @@ def llm_inference(
     return df
 
 
-def parse_llm_output(llm_output: str) -> Union[Dict, Exception]:
+def parse_llm_output(llm_output: str | Exception) -> Union[Dict, Exception]:
```

Comment:
Can we change this to:

`def parse_llm_output(llm_output: Union[str, Exception]) -> Union[Dict, Exception]:`


because the syntax you provided is not supported in older python versions like 3.9